djangocms_references/cms_config.py

```diff
@@ -110,11 +117,32 @@ def version_queryset_modifier(queryset):
     return queryset
 
 
+def unpublish_dependencies(request, version, *args, **kwargs):
+    """Render a partial template with a list of unpublish dependencies"""
+    references = get_all_reference_objects(
+        version.content, draft_and_published=True)
```

Comment:
Why "draft_and_published"? Seems counter-intuitive... as in we're searching for unpublished items not items that are either draft or published.

Reply:
We're actually not searching for unpublished items. We're searching for items that have relationships with an item we're unpublishing.